### ISSUE TITLE
👨‍🔧Ensure assertion works in iframe

### ISSUE THREAD

**Issue by jperl** [qawolf/qawolf#137]
@flaurida ran into an issue recording a test for
https://www.printingforless.com with this assertion

```
const hasBuckSlipsText = await hasText("Buck Slips");
expect(hasBuckSlipsText).toBe(true);
```

UPDATE: Issue is that is in an iframe. Will retry after #122 is resolved

**Comment by jperl** [comment]
Issue is that is in an iframe. To be resolved in https://github.com/qawolf/qawolf/issues/122

**Comment by jperl** [comment]
Closing as duplicate of #122
